### PR TITLE
Tomun/fixflow

### PR DESCRIPTION
**Pull request by tom-un**
#### Please select one of the following
- [ ] I am removing an existing difference between facebook/react-native and microsoft/react-native :thumbsup:
- [ ] I am cherry-picking a change from Facebook's react-native into microsoft/react-native :thumbsup:
- [X] I am making a fix / change for the macOS implementation of react-native
- [ ] I am making a change required for Microsoft usage of react-native

The Flow checks have been broken in the Microsoft react-native fork for some time.   This change adds Flow, Jest, Lint, and Prettier tests to the PR tests.  It also fixes Flow breaks that have been in the repo since the last merge from .59 to .60.

While fixing and testing the Flow changes I discovered that the mac keyboard scroll events were not working.   This change also restores the mac keyboard scroll events (scroll by using arrow keys, etc.).   The mac keyboard scrolling depends on a native ScrollView prop `onKeyDown`.  This can easily be confused with generic key events so I renamed `onKeyDown` to `onScrollKeyDown`.

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native/pull/196)

### CODE REVIEW

File: Libraries/Components/ScrollView/ScrollView.js [hunk 1]
```diff
@@ -883,58 +882,58 @@ class ScrollView extends React.Component<Props, State> {
...
 
   // [TODO(macOS ISS#2323203)
-  _handleKeyDown = (e: KeyboardEvent) => {
-    if (this.props.onKeyDown) {
-      this.props.onKeyDown(e);
+  _handleKeyDown = (e: ScrollEvent) => {
+    if (this.props.onScrollKeyDown) {
+      this.props.onScrollKeyDown(e);
     } else {
       const event = e.nativeEvent;
```
Comment (anandrajeswaran) [inline Libraries/Components/ScrollView/ScrollView.js]: Probably beyond scope (adds more unnecessary diff?) to rename this to nativeEvent and "e" to "event"?

Comment (tom-un) [reply Libraries/Components/ScrollView/ScrollView.js]: no problem
